pkg/daemon/ceph/mds/filesystem.go

```diff
@@ -71,15 +69,24 @@ func (f *Filesystem) CreateFilesystem(context *clusterd.Context, clusterName str
 		return fmt.Errorf("at least one data pool must be specified")
 	}
 
-	err = ceph.CreatePoolWithProfile(context, clusterName, *f.metadataPool, appName)
+	fslist, err := client.ListFilesystems(context, clusterName)
+	if err != nil {
+		return fmt.Errorf("Unable to list existing file systems. %+v", err)
+	}
+	if len(fslist) > 0 && !(client.IsMultiFSEnabled()) {
+		return fmt.Errorf("Cannot create multiple filesystems. Enable ROOK_ALLOW_MULTIPLE_FILESYSTEMS env variable to create more than one")
```

Comment:
could you format in the env variable name to the message?

Reply:
would be good to have a unit tests that verifies this error is returned when the env variable is set. Looks like the `pkg/operator/file/ceph` package already has tests in place to add to

Reply:
Thanks for the tip. First time trying out go tests so bear with me on the tardiness of the work, if any.